cdap-cli/bin/cdap-cli.sh

```diff
@@ -34,14 +34,17 @@ done
 bin=`dirname "${PRG}"`
 bin=`cd "$bin"; pwd`
 lib="$bin"/../lib
+libexec="$bin"/../libexec
 conf="$bin"/../conf
 CDAP_CONF=${CDAP_CONF:-/etc/cdap/conf}
 script=`basename $0`
 
+CLI_CP=${libexec}/co.cask.cdap.cdap-cli-@@project.version@@.jar:${lib}/co.cask.cdap.cdap-cli-@@project.version@@.jar
```

Comment:
The script is shared by SDK and RPM/DEB packaging. In distributed mode, those jars are still copied to "lib", but it's under `/opt/cdap/cli/lib`, which doesn't affect how master operates.